monitoring/ceph-mixin/prometheus_alerts.libsonnet

```diff
@@ -736,5 +736,50 @@
         },
       ],
     },
+    {
+      name: 'rbdmirror',
+      rules: [
+        {
+          alert: 'CephRBDMirrorImagesPerDaemonHigh',
+          'for': '1m',
+          expr: 'sum by (ceph_daemon, namespace) (ceph_rbd_mirror_snapshot_image_snapshots) > %(CephRBDMirrorImagesPerDaemonThreshold)s' % $._config,
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.10.2' },
+          annotations: {
+            summary: 'Number of image replication is now above %(CephRBDMirrorImagesPerDaemonThreshold)s' % $._config,
+            description: 'Number of image replication per daemon is not suppossed to go beyond the provided threshold',
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesNotInSync',
+          'for': '1m',
+          expr: 'sum by (ceph_daemon, image, namespace, pool) (topk by (ceph_daemon, image, namespace, pool) (1, ceph_rbd_mirror_snapshot_image_local_timestamp) - topk by (ceph_daemon, image, namespace, pool) (1, ceph_rbd_mirror_snapshot_image_remote_timestamp)) != 0',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.10.3' },
+          annotations: {
+            summary: 'Some of the RBD mirror images are not in sync with the remote counter parts.',
+            description: 'Both local and remote RBD mirror images should be in sync.',
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImagesNotInSyncVeryHigh',
+          'for': '1m',
+          expr: 'count by (ceph_daemon) ((topk by (ceph_daemon, image, namespace, pool) (1, ceph_rbd_mirror_snapshot_image_local_timestamp) - topk by (ceph_daemon, image, namespace, pool) (1, ceph_rbd_mirror_snapshot_image_remote_timestamp)) != 0) > (sum by (ceph_daemon) (ceph_rbd_mirror_snapshot_snapshots)*.1)',
+          labels: { severity: 'critical', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.10.4' },
+          annotations: {
+            summary: 'Number of unsynced images are very high.',
+            description: 'Very high number of unsynced images detected.',
+          },
+        },
+        {
+          alert: 'CephRBDMirrorImageTransferBandwidthHigh',
+          'for': '1m',
+          expr: 'rate(ceph_rbd_mirror_journal_replay_bytes[5m]) > %(CephRBDMirrorImageTransferBandwidthThreshold)s' % $._config,
+          labels: { severity: 'warning', type: 'ceph_default', oid: '1.3.6.1.4.1.50495.1.2.1.10.5' },
+          annotations: {
+            summary: 'A very high network usage detected due to image transfers. Please reduce the number of images to replicate.',
```

Comment:
I think, is good to have this alert, but we cannot deem it as an error, so the description must be more precise. For example, imagine that we have not started replication, and we start to replicate 1 image, we are going to pass from 0 to "something" in "ceph_rbd_mirror_journal_replay_bytes" and the alert is going to raise.

To avoid false positives i will increase the rate time from 5 min to 30 min... 

I will use this summary and description:
Summary: 'The replication network usage has been increased over 80% in the last 30 minutes. Review the number of images being replicated. This alert will be cleaned automatically after 30 minutes"
Description: 'It has been detected a heavy increase of the bandwidth needed for rbd replication (over 80%) in the last 30 min. This might be not a problem, but is good to review the number of images replicating simultaneously"